File: bronze/8958/python/main.py

if (__name__ == "__main__"):
    num = int(input())

    for i in range(num):
        question = input()
        result, count = (0, 0)
        for data in question:
            count = (count +1) if (data == 'O') else 0
            result += count
        print(result)

File: bronze/8958/c++/main.cpp
#include <iostream>
#include <string>

int main()
{
    int n, result, count;
    std::cin >> n;
    
    std::string question;
    for (int i = 0; i < n; i++) {
        std::cin >> question;
        result = 0, count = 0;
        for (int j = 0; j < question.size(); j++) {
            count = (question[j] == 'O') ? count + 1 : 0;
            result += count;
        }
        std::cout << result << std::endl;

    }
    return 0;
}